events/wmt21.md

```diff
@@ -217,7 +217,7 @@ Thursday, November 11, 2021
 ### News Translation
 
 #### → English (segment rating with document context)
-> *Systems ordered by Direct Assessment score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05>*
```

Comment:
dear @cefoo, I've also deleted some parts of the description as I've just realized that those are not relevant if we show only winners and not all the results